### PR TITLE
StaveTie updated so that direction can be set, else falls back to normal...

### PR DESCRIPTION
**Pull request by jmahmud**
... assignement from first/last notes

### CODE REVIEW

**Comment by jmahmud** [comment]
Just a few minor suggestions: main change is in the stavehairpin.js where I've added a beginPath & closePath (mimicking how crescendo.js works) as without it, was cause a duplication of surrounding glyphs.

File: tests/flow.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <title>VexFlow 2 - JavaScript Music Notation and Guitar Tab</title>
   <link rel="stylesheet" href="flow.css" type="text/css" media="screen" />
   <!-- VexFlow Sources -->
-  <script src="../build/vexflow-debug.js"></script>
+  <script src="../releases/vexflow-debug.js"></script>
```
Comment (0xfe) [inline tests/flow.html]: Is this line intentional?

**Comment by 0xfe** [comment]
Change looks good. Can you add tests please?

**Comment by jmahmud** [comment]
HI Mohit - will definitely add tests and also see if I can do something with those hardcoded values in the stavehairpin class.  

Cheers.
Josh

File: tests/flow.html (hunk 1 above)
Comment (jmahmud) [inline tests/flow.html]: Yes - it seems that the built vexflox-debug.js has move so I have updated the src...flow.html stopped working without this change - did I miss something perhaps?

Comment (0xfe) [inline tests/flow.html]: Running `grunt` will put files in `build/`. Also, `grunt watch` will keep `build/` updated as you change sources. The `releases/` directory is what I populate when I cut a new release and publish and NPM.

**Comment by SalahAdDin** [comment]
:+1:

**Comment by 0xfe** [comment]
Thanks for these changes, and for adding the tests. Merged.

**Pull request merged**